lib/mix/test/mix/tasks/escript_test.exs

```diff
@@ -97,6 +108,19 @@ defmodule Mix.Tasks.EscriptTest do
     end
   end
 
+  test "generate escript with debug information" do
```

Comment:
I don't think we can do it from internally because the beams are always in memory, so trying to fetch anything from beam won't work. :(